compiler/optimizer/CopyPropagation.cpp

```diff
@@ -745,16 +745,18 @@ int32_t TR_CopyPropagation::perform()
    for (auto itr = usesToBeFixed.begin(), end = usesToBeFixed.end(); itr != end; )
       {
       int32_t fixUseIndex = itr->first;
+      TR::Node *fixUseNode = useDefInfo->getNode(fixUseIndex);
       invalidateDefUseInfo = true;
+      if (useDefInfo->hasLoadsAsDefs())
```

Comment:
Is this condition really needed ? I ask, because it is not used before the later loop in this same commit. I'd have expected us to be consistent. Or is there some subtle reason why the first spot is different ?